torch/_prims/utils.py

```diff
@@ -930,12 +930,16 @@ def compute_reduction_output_shape(
     return tuple(new_shape)
 
 
+def validate_no_repeating_dims(dims: Sequence):
```

Comment:
nit: we should update the typing here and on `reduction_dims` to use DimsSequenceType